application/genesisrefs/contracts/contracts.go

```diff
@@ -30,143 +32,131 @@ import (
 	"github.com/insolar/insolar/application/builtin/contract/rootdomain"
 	"github.com/insolar/insolar/application/builtin/contract/wallet"
 
-	"github.com/insolar/insolar/application"
 	"github.com/insolar/insolar/application/appfoundation"
 	maProxy "github.com/insolar/insolar/application/builtin/proxy/migrationshard"
 	pkProxy "github.com/insolar/insolar/application/builtin/proxy/pkshard"
-	"github.com/insolar/insolar/application/genesisrefs"
 	"github.com/insolar/insolar/insolar"
 	"github.com/insolar/insolar/logicrunner/builtin/foundation"
 	"github.com/insolar/insolar/pulse"
 )
 
-func RootDomain(pkShardCount int) application.GenesisContractState {
+func RootDomain(pkShardCount int) genesis.GenesisContractState {
 
-	return application.GenesisContractState{
+	return genesis.GenesisContractState{
 		Name:       application.GenesisNameRootDomain,
 		Prototype:  application.GenesisNameRootDomain,
 		ParentName: "",
 
-		Memory: mustGenMemory(&rootdomain.RootDomain{
+		Memory: genesis.MustGenMemory(&rootdomain.RootDomain{
 			PublicKeyShards: genesisrefs.ContractPublicKeyShards(pkShardCount),
 		}),
 	}
 }
 
-func NodeDomain() application.GenesisContractState {
-	nd, _ := nodedomain.NewNodeDomain()
-	return application.GenesisContractState{
-		Name:       application.GenesisNameNodeDomain,
-		Prototype:  application.GenesisNameNodeDomain,
-		ParentName: application.GenesisNameRootDomain,
-		Memory:     mustGenMemory(nd),
-	}
-}
-
-func GetMemberGenesisContractState(publicKey string, name string, parent string, walletRef insolar.Reference) application.GenesisContractState {
+func GetMemberGenesisContractState(publicKey string, name string, parent string, walletRef insolar.Reference) genesis.GenesisContractState {
```

Comment:
we can remove `Genesis` part from all names inside genesis package to avoid redundancy:
i.e. `GetMemberGenesisContractState` -> `GetMemberContractState`